### ISSUE TITLE
problems with reconection

### ISSUE THREAD

**Issue by VictorHVS** [nlsdfnbch/btfxwss#160]
I got this error:

2018-09-24 18:32:36,645 - btfxwss.connection - INFO - Connection opened
2018-09-24 18:32:36,647 - btfxwss.connection - INFO - Connection Error - filedescriptor out of range 
in select()
2018-09-24 18:32:36,647 - websocket - ERROR - close status: 31522
2018-09-24 18:32:36,648 - btfxwss.connection - INFO - Connection is closed normally.
2018-09-24 18:32:37,202 - btfxwss.connection - INFO - Connection opened
2018-09-24 18:32:37,204 - btfxwss.connection - INFO - Connection Error - filedescriptor out of range in select()
2018-09-24 18:32:37,204 - websocket - ERROR - close status: 31522
2018-09-24 18:32:37,205 - btfxwss.connection - INFO - Connection is closed by Bitfinex.
2018-09-24 18:32:37,208 - btfxwss.connection - INFO - Attempting to connect again in 10 seconds.
2018-09-24 18:32:47,750 - btfxwss.connection - INFO - Connection opened
2018-09-24 18:32:47,751 - btfxwss.connection - INFO - _on_open(): Connection reconnected, re-subscribing..
2018-09-24 18:32:47,751 - btfxwss.connection - INFO - Connection Error - filedescriptor out of range in select()
2018-09-24 18:32:47,751 - websocket - ERROR - close status: 31522
2018-09-24 18:32:47,752 - btfxwss.connection - INFO - Connection is closed by Bitfinex.
2018-09-24 18:32:47,754 - btfxwss.connection - INFO - Attempting to connect again in 10 seconds.
2018-09-24 18:32:59,013 - btfxwss.connection - INFO - Connection opened
2018-09-24 18:32:59,014 - btfxwss.connection - INFO - _on_open(): Connection reconnected, re-subscribing..
2018-09-24 18:32:59,014 - btfxwss.connection - INFO - Connection Error - filedescriptor out of range in select()
2018-09-24 18:32:59,015 - websocket - ERROR - close status: 31522
2018-09-24 18:32:59,015 - btfxwss.connection - INFO - Connection is closed by Bitfinex.
2018-09-24 18:32:59,018 - btfxwss.connection - INFO - Attempting to connect again in 10 seconds.

**Comment by deepbrook** [comment]
Does this issue persist for you? Bitfinex's websocket sometimes is a bit..peculiar.

**Comment by VictorHVS** [comment]
Yes, it persists. Why don't you use this: https://websockets.readthedocs.io/en/stable/intro.html? I'm using and no problems so far.

**Comment by deepbrook** [comment]
I'm currently migrating the library. However, this is a pastime project and I simply don't have enough time to put in as much work as I'd like.
